### PR TITLE
fix readme examples fail

### PR DESCRIPTION
**Pull request by intech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -501,7 +501,7 @@ v.validate({ mac: "01C8.954B.65FE" }, schema); // Valid
 v.validate({ mac: "01c8.954b.65fe", schema); // Valid
 v.validate({ mac: "01-C8-95-4B-65-FE" }, schema); // Valid
 v.validate({ mac: "01-c8-95-4b-65-fe" }, schema); // Valid
-v.validate({ mac: "01:C8:95:4B:65:FE" }, schema); // Fail
+v.validate({ mac: "01C8954B65FE" }, schema); // Fail
 ```
```
Comment (icebob) [inline README.md]: It means `"01:C8:95:4B:65:FE"` is valid? If yes, please also add it as valid example.

Comment (intech) [reply README.md]: > It means `"01:C8:95:4B:65:FE"` is valid? If yes, please also add it as valid example.

it is the very first line, in this diff is not visible

Comment (intech) [reply README.md]: @icebob https://github.com/icebob/fastest-validator/commit/88675f8257386dac153df06c758e8362221029ab#diff-04c6e90faac2675aa89e2176d2eec7d8R498

Comment (icebob) [reply README.md]: Sorry, yes, diff doesn't show it.

**Pull request merged**